### PR TITLE
Fix broken link

### PR DESCRIPTION
**Pull request by itaileshem**
#### Summary
Fixed a broken link

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ x ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/javascript/memory_management/index.md [hunk 1]
```diff
@@ -205,4 +205,4 @@ node --expose-gc --inspect index.js
 
 - [IBM article on "Memory leak patterns in JavaScript" (2007)](https://www.ibm.com/developerworks/web/library/wa-memleak/wa-memleak-pdf.pdf)
 - [Kangax article on how to register event handler and avoid memory leaks (2010)](https://msdn.microsoft.com/en-us/magazine/ff728624.aspx)
-- [Performance](/en-US/docs/Mozilla/Performance)
+- [Performance](/en-US/docs/Web/API/Performance)
```
Comment (wbamberg) [inline files/en-us/web/javascript/memory_management/index.md:208]:
```suggestion
```

**Review by wbamberg — CHANGES_REQUESTED** [review]
Well, spotted, and thanks for filing this PR! It seems to me though that this link isn't relevant to the page at all, shall we just remove it?

**Comment by itaileshem** [comment]
> Well, spotted, and thanks for filing this PR! It seems to me though that this link isn't relevant to the page at all, shall we just remove it?

Yea, it does feel a little out of place.

**Review by wbamberg — APPROVED** [review]
Thanks for your contribution @itaileshem !

**Pull request merged**